src/endpoint/iam/iam_constants.js

```diff
@@ -18,20 +15,58 @@ const IAM_ACTIONS = Object.freeze({
     LIST_ACCESS_KEYS: 'list_access_keys'
 });
 
+// key: action - the function name on accountspace_fs (snake case style)
+// value: AWS action name (camel case style)
+// we use it for error message to match AWS style 
+const ACTION_MESSAGE_TITLE_MAP = Object.freeze({
+    'create_user': 'CreateUser',
+    'get_user': 'GetUser',
+    'delete_user': 'DeleteUser',
+    'update_user': 'UpdateUser',
+    'list_users': 'ListUsers',
+    'create_access_key': 'CreateAccessKey',
+    'get_access_key_last_used': 'GetAccessKeyLastUsed',
+    'update_access_key': 'UpdateAccessKey',
+    'delete_access_key': 'DeleteAccessKey',
+    'list_access_keys': 'ListAccessKeys',
+});
 
-// parameter names
-const IAM_PATH = 'Path';
-const NEW_IAM_PATH = 'NewPath';
-const IAM_PATH_PREFIX = 'PathPrefix';
-const USERNAME = 'UserName';
-const NEW_USERNAME = 'NewUserName';
+const access_key_status_enum = Object.freeze({
```

Comment:
minor: I would make it upper case. these are just nested constants, but still constants.

Reply:
I can change it, I was inspired by this enum:
https://github.com/noobaa/noobaa-core/blob/909e29f92ad0d65fe65a9176058d62444b843fb8/src/sdk/namespace_fs.js#L71-L75

Reply:
yeah but i think we should be consistent to define constants with uppercase and Object.freeze() - this is a (semi)standard across many languages, and helps readability.